Resources/Prototypes/_RMC14/Roles/Jobs/Other/senior_enlisted_advisor.yml

```diff
@@ -87,7 +87,7 @@
   id: CMGuidebookRoleSEA
   name: Senior Enlisted Advisor
   description: Mentor for the Marines!
-  noSpawn: false
+  categories: [ HideSpawnMenu ]
```

Comment:
as i commented in the issue, its nospawn false not true, so set categories to []

Reply:
This shouldn't be spawnable, as it's a guidebook entity I created, is that not the correct usage?